ranker/shared/ranker_obj.py

```diff
@@ -566,8 +566,12 @@ def get_edge_support_kg(edge_id, kg, aux_graphs, edge_kg=None):
                 for add_edge_id in sg_edges:
                     # Get this edge and add it to the edge_kg
                     edge_kg['edge_ids'].add(add_edge_id)
```

Comment:
Do we want to add the edge id to this list even if it doesn't exist in the kg?

Reply:
Good point.   See new commit.